exercises/sum-of-multiples/example.go

```diff
@@ -1,16 +1,15 @@
 package summultiples
 
-func MultipleSummer(divisors ...int) func(int) int {
-	return func(limit int) int {
-		sum := 0
-		for i := 1; i < limit; i++ {
-			for _, d := range divisors {
-				if i%d == 0 {
-					sum += i
-					break
-				}
+// MultipleSummer returns the sum of the multiples of the given divisors
+// up to, but not including, the given limit.
+func MultipleSummer(limit int, divisors ...int) (sum int) {
```

Comment:
Yeah, `summultiples.SumMultiples` isn't perfect but will do for now, no complaints from `go lint`